### ISSUE TITLE
Quicktime .MOV support

### ISSUE THREAD

**Issue by foliovision** [foliovision/freedom-video-player#400]
Original issue link: [#544 Quicktime .MOV support](https://github.com/flowplayer/flowplayer/issues/544)

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/820962?v=4' /> [andheiberg](https://github.com/andheiberg) commented on Sep 30, 2013

My users are uploading straight from their iPhones and I'm trying to avoid any conversion. I incorrectly had these files marked as 'video/mp4' in my html and it seamed to work great.

However I changed it to 'video/quicktime'  after discovering that my files where sent from the server with 'video/quicktime'. This however results in an error message in the player "unsupported video format" I think.

Why is .MOV not supported?

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/168338?v=4' /> [phloxic](https://github.com/phloxic) commented on Sep 30, 2013

Have a look here with Chrome: http://demos.flowplayer.org/videotest/canplay.html - Chrome will not accept video/quicktime in html5 video.
In short: ignore the filename suffix and mov container format, if you're confident that the videos are encoded with the right codecs and settings - which is not always the case when the movies come from iPhones, they might have not set their display and sample aspect ratio set, which can lead to problems in IE9 and IE10.

Finally: please turn to our forum at http://flowplayer.org/forum first - please read: https://github.com/flowplayer/flowplayer/blob/master/CONTRIBUTING.md#reporting-issues
